lib/SIL/SILFunctionType.cpp

```diff
@@ -2496,6 +2456,46 @@ static CanType copyOptionalityFromDerivedToBase(TypeConverter &tc,
   return base;
 }
 
+// SWIFT_ENABLE_TENSORFLOW
+// TODO: Move this to a common location (SILFunctionType.cpp).
+static CanSILFunctionType
```

Comment:
Add a comment. What does “normalize” mean?

Reply:
Added comment in https://github.com/apple/swift/pull/26709/commits/1093416710d66105e8dc09a6110aed839b535975.